index.html

```diff
@@ -62,7 +67,9 @@ <h2>My Recent Works</h2>
             <div class="line">
                 <img src="./images/greenline.png" alt="greenline">
             </div>
+        </section>
 
+        <section class="my-projects">
             <div class="project-container">
                     <div class="single-project">
                         <img src="./images/placeholder.png" alt="placeholder">
```

Comment:
- [ ] Your projects section isn't currently responsive because of the hard-coded width values. Please try making it as responsive as possible, by using `display:flex` to position the elements in the work cards, and removing the hard-coded width pixels.

Here's what it currently looks like versus what it looks like with just a few changes:


![](https://i.imgur.com/8bl00WI.png)



![](https://i.imgur.com/jKsVyxT.png)